### PR TITLE
1276 reverting change to ExternalAccountStoreModelFactory

### LINKED ISSUE #1276: AccountStoreVisitorAdapter must not throw
`AccountStoreVisitorAdapter` should be changed to not throw on each method.

Additionally:
1. [This commit](https://github.com/stormpath/stormpath-sdk-java/commit/74dbf117243db4619628c83a668b76c9403df50c#diff-63341606beae5c59bff35881f09ca9f8) must be undone.
2. Check that `AccountStoreVisitorAdapter` is not being used anywhere else and that this change is not impacted it

### PR DESCRIPTION
**Pull request by mrioan**
Resolves #1276 

This PR tries to revert this: https://github.com/stormpath/stormpath-sdk-java/commit/74dbf117243db4619628c83a668b76c9403df50c#diff-63341606beae5c59bff35881f09ca9f8

### CODE REVIEW

File: extensions/servlet/src/main/java/com/stormpath/sdk/servlet/mvc/provider/ExternalAccountStoreModelFactory.java [hunk 1]
```diff
@@ -78,7 +80,7 @@ protected String getAuthorizeBaseUri(@SuppressWarnings("UnusedParameters") HttpS
         return authorizeBaseUri;
     }
 
-    private class AccountStoreModelVisitor extends AccountStoreVisitorAdapter {
```
Comment (palimpsestor) [inline extensions/servlet/src/main/java/com/stormpath/sdk/servlet/mvc/provider/ExternalAccountStoreModelFactory.java]: Kind of surprised that you switched to no-ops in the `AccountStoreVisitorAdapter` but then decided not to use it here but instead reimplement the no-ops (for the Group and Organization cases).

Comment (mrioan) [inline extensions/servlet/src/main/java/com/stormpath/sdk/servlet/mvc/provider/ExternalAccountStoreModelFactory.java]: @palimpsestor, you are totally right to be surprised! For the sake of re-use I will use the adapter once again! Dumb me... Thanks @palimpstor

Comment (mrioan) [inline extensions/servlet/src/main/java/com/stormpath/sdk/servlet/mvc/provider/ExternalAccountStoreModelFactory.java]: fixed

**Comment by mrioan** [comment]
hey @palimpsestor Is this PR approved? Thanks!

**Pull request merged**